include/Track.h

```diff
@@ -226,13 +228,18 @@ class TrackContentObjectView : public selectableObject, public ModelView
 	QColor textShadowColor() const;
 	QColor BBPatternBackground() const;
 	bool gradient() const;
+	int getMouseHotspotX() const { return m_mouseHotspotX; }
+	int getMouseHotspotY() const { return m_mouseHotspotY; }
```

Comment:
Are these functions necessary? I removed all references to `getMouseHotspot_`, and the PR still compiled and worked as expected. Once again, I'm concerned about adding more cursors, so it'd be ideal if each one took as few lines as possible.

Reply:
Indeed, I removed those here and after the `READ` in the properties. Works fine.